libdnf/dnf-context.c

```diff
@@ -1027,6 +1044,22 @@ dnf_context_set_cache_age(DnfContext *context, guint cache_age)
     priv->cache_age = cache_age;
 }
 
+
+/**
+ * dnf_context_set_cache_suffix_match:
+ * @context: a #DnfContext instance.
+ * @cache_suffix_match : %FALSE means the cache needs to be re-updated (default is false)
+ *
+ * Sets status of cache suffix
+ *
+ **/
+void
+dnf_context_set_cache_suffix_match(DnfContext *context, gboolean cache_suffix_match)
```

Comment:
Why do we need these methods? I don't think this is something we want clients to be able to modify, right? Can we just drop `cache_suffix_match` and the methods completely?